.husky/pre-commit

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+echo "Linting your code, please wait⏳\n"
```

Comment:
Shouldn't this be `Checking for linting problems, please wait`?
What do you think?